veggieseasons/lib/screens/details.dart

```diff
@@ -13,9 +13,10 @@ import 'package:veggieseasons/styles.dart';
 /// harvested. It displays the first two letters of the season and uses a
 /// different background color to represent each of the seasons as well.
 class SeasonCircle extends StatelessWidget {
+  final bool active;
```

Comment:
dartdoc it. Just looking at it, I can't tell if it's the current season or the focused season.

Reply:
Renamed and better explained.